src/routes/Selection/Selection.tsx

```diff
@@ -70,83 +85,131 @@ interface SelectionProps {
   selection: SelectionType;
   checkPopulationRegistraion: typeof checkPopulationRegistraion;
   deleteSelection: typeof deleteSelection;
+  sortSelection: typeof sortSelection;
 }
 
 const Selection: React.FC<SelectionProps> = ({
   selection,
   deleteSelection,
   checkPopulationRegistraion,
-}) => (
-  <>
-    <BackButton to="/" />
-    <h1>{selection.name}</h1>
-    <Header>
-      <div>
-        <div>
-          <strong>Sökterm:</strong> {selection.selection_term}
-        </div>
+  sortSelection,
+}) => {
+  const [sortOptions] = useState({
```

Comment:
This don't need to be a state since we never update it.

Reply:
Yes we do, in the sort function.

Reply:
Sorry didn't get a notification about this. But if/when you update it you should updating it by using setSortOption. And pass it down to the sort fn aswell.